test/puppetlabs/puppetdb/testutils/http.clj

```diff
@@ -48,7 +48,10 @@
      (is (= http/status-ok (:status response)))
      (-> response
          :body
-         slurp
+         ((fn [response-body]
+            (if (string? response-body)
+              response-body
+              (slurp response-body))))
          (json/parse-string true)
```

Comment:
I think this would be nicer if we split it out into it's own function -- maybe-slurp or something.